src/compile_config.h

```diff
@@ -21,19 +21,26 @@
 */
 
 #ifdef OTHER_BOARD_WITH_CC1101
-#define CMP_CC1101
+	#define CMP_CC1101
 #endif
 #ifdef ARDUINO_ATMEGA328P_MINICUL
-#define CMP_CC1101
+	#define CMP_CC1101
 #endif
 
 // Get compatibility with arduino ide and visualmicro
 #ifdef ARDUINO_AVR_ICT_BOARDS_ICT_BOARDS_AVR_RADINOCC1101
-#define ARDUINO_RADINOCC1101
+	#define ARDUINO_RADINOCC1101
 #endif
 
 #ifdef ARDUINO_RADINOCC1101
-#define CMP_CC1101     
+	#define CMP_CC1101
+#endif
+
+#if defined(ARDUINO_MAPLEMINI_F103CB) || defined(STM32F103xB)          // STM32 F103CBT6
+	#define MAPLE_Mini 1
+	#ifdef MAPLE_WATCHDOG
```

Comment:
Den Sinn hier verstehe ich nicht, etwas das definiert ist, mit dem Wert 1 erneut zu definieren. Ist dir das transparent?

Reply:
Das Verhalten kam ebenso auf mit 2 Plattformen in der Entwicklung.

Wenn du den Wert 1 weg lässt, so kann das PlatformIO nicht kompilieren.
Arduino schlägt dann auch fehl, Bsp an einer Abfrage
`	#elif MAPLE_Mini`

Ich habe es soeben mal provoziert und somit kommt ein Fehler.